nav2_rotation_shim_controller/src/nav2_rotation_shim_controller.cpp

```diff
@@ -391,11 +396,11 @@ bool RotationShimController::isGoalChanged(const nav_msgs::msg::Path & path)
   return current_path_.poses.back().pose != path.poses.back().pose;
 }
 
-void RotationShimController::setPlan(const nav_msgs::msg::Path & path)
+void RotationShimController::newPathReceived(const nav_msgs::msg::Path & raw_global_path)
 {
-  path_updated_ = rotate_to_heading_once_ ? isGoalChanged(path) : true;
-  current_path_ = path;
-  primary_controller_->setPlan(path);
+  path_updated_ = rotate_to_heading_once_ ? isGoalChanged(raw_global_path) : true;
+  current_path_ = raw_global_path;
```

Comment:
Should we use the new `computeVelocityCommands` for this instead for setting the current path?